README.md

```diff
@@ -1,31 +1,24 @@
-# The Haskell Standard Library
-
-## Random Number Generation
-
-### Status
-
-| Language | Travis | Coveralls |
-|:--------:|:------:|:---------:|
-| ![GitHub top language](https://img.shields.io/github/languages/top/idontgetoutmuch/random.svg) | [![Build Status](https://secure.travis-ci.org/idontgetoutmuch/random.svg?v1.2-proposal)](http://travis-ci.org/idontgetoutmuch/random) | [![Coverage Status](https://coveralls.io/repos/github/idontgetoutmuch/random/badge.svg?branch=v1.2-proposal)](https://coveralls.io/github/idontgetoutmuch/random?branch=v1.2-proposal)
-
-|      Package       | Hackage | Nightly | LTS |
-|:-------------------|:-------:|:-------:|:---:|
-|  [`random`](https://github.com/idontgetoutmuch/random)| [![Hackage](https://img.shields.io/hackage/v/random.svg)](https://hackage.haskell.org/package/random)| [![Nightly](https://www.stackage.org/package/random/badge/nightly)](https://www.stackage.org/nightly/package/random)| [![Nightly](https://www.stackage.org/package/random/badge/lts)](https://www.stackage.org/lts/package/random)
-
-
-### Description
-
-This library provides a basic interface for (splittable) random number generators.
-
-The API documentation can be found here:
-
-   http://hackage.haskell.org/package/random/docs/System-Random.html
-
-A module supplying this interface is required for Haskell 98 (but not Haskell
-2010). An older [version]
-(http://www.haskell.org/ghc/docs/latest/html/libraries/haskell98/Random.html)
-of this library is included with GHC in the haskell98 package. This newer
-version, with compatible api, is included in the [Haskell Platform]
-(http://www.haskell.org/platform/contents.html).
-
-Please report bugs in the Github [issue tracker] (https://github.com/haskell/random/issues) (no longer in the GHC trac).
+# random
+
+[![Build Status](https://secure.travis-ci.org/idontgetoutmuch/random.svg?v1.2-proposal)](http://travis-ci.org/idontgetoutmuch/random)
+[![Coverage Status](https://coveralls.io/repos/github/idontgetoutmuch/random/badge.svg?branch=v1.2-proposal)](https://coveralls.io/github/idontgetoutmuch/random?branch=v1.2-proposal)
+
+**Pseudo-random number generation for Haskell.**
+
+This library provides
+- an interface for pure and optionally splittable pseudo-random number
+  generators: [`RandomGen`][haddock-randomgen]
+- an interface for monadic pseudo-random number generators:
+  [`MonadRandom`][haddock-monadrandom]
+- adapters to use pure pseudo-random number generators in monadic code: [see
+  documentation][haddock-adapters]
+- a pure pseudo-random number generator implementation that is fast,
+  splittable, and passes empirical tests: [`StdGen`][haddock-stdgen]
```

Comment:
It would of course be nice to point to evidence for "passes empirical tests", but I don't know how best to do that.

Perhaps there should be a little summary for SplitMix in the `results` directory of https://github.com/tweag/random-quality/ that we can point to?

This is just me thinking aloud - I think this can be addressed as a follow-up.